projects/1.16/assets/hats/hats/lang/zh_cn.json

```
@@ -1 +1,244 @@
-{}
\ No newline at end of file
+{
+  "config.hats.cat.hatRandomisation.name": "帽子随机化",
+  "config.hats.cat.hatRandomisation.desc": "帽子随机化选项",
+  "config.hats.cat.others.name": "其他选项",
+  "config.hats.cat.others.desc": "不属于任何地方的选项。",
+
+  "config.hats.prop.maxHatRenders.name": "帽子最大渲染数量",
+  "config.hats.prop.maxHatRenders.desc": "每帧最多渲染的帽子数量",
+  "config.hats.prop.hatChance.name": "帽子生成概率",
+  "config.hats.prop.hatChance.desc": "对于客户端配置：在未安装帽子模组的服务器上，实体佩戴帽子的概率。\n对于服务器配置：支持佩戴帽子的实体佩戴帽子的概率。",
+  "config.hats.prop.hatUnlockString.name": "帽子解锁消息",
+  "config.hats.prop.hatUnlockString.desc": "解锁帽子、配饰时，屏幕提示的标题。0=随机，1=帽子已解锁，2=新帽子，3=帽子已获得，4=帽子已收集，5=得到了帽子",
+  "config.hats.prop.guiMinecraftStyle.name": "以Minecraft风格渲染GUI",
+  "config.hats.prop.guiMinecraftStyle.desc": "我们是否以Minecraft风格渲染GUI（而不是界面和其他东西）？1=原版风格，2=材质包风格",
```

```suggestion
  "config.hats.prop.guiMinecraftStyle.desc": "是否以Minecraft风格渲染GUI（而不是界面和其他东西）？1=原版风格，2=材质包风格",
```
这个boxes & stuff真没看懂（）